examples/nlp/language_modeling/tuning/conf/megatron_gpt_peft_eval_config.yaml

```diff
@@ -105,8 +105,7 @@ model:
       drop_last: False
       context_key: ${data.train_ds.context_key}
       label_key: ${data.train_ds.label_key}
-      query_key: null # This key is used when context is too long, and we first want to truncate context and then append query to it.
-      add_eos: ${data.train_ds.add_eos}
+      add_eos: False
```

Comment:
Validation and test set don't need eos token because they start generation and we don't need to add them. 
I previously check this directly in the dataset. However, I think it would be better just change in config.